bindings/rust/aws-lc-sys-template/LICENSE

```diff
@@ -0,0 +1,456 @@
+AWS Lib Crypto is a fork of BoringSSL, which is itself a fork of OpenSSL. New 
```

Comment:
Can this reference our top level license file to avoid this duplicate?

Reply:
This file will be published as part of the crate. I could copy the LICENSE from the top-level during crate generation instead of placing it in this "template" -- it's just more convenient having it in the template.

Reply:
I agree.  I'll remove this license, and have the script copy from the top-level.